src/App.js

```diff
@@ -1,19 +1,45 @@
 import React from 'react';
+import { Tabs } from './components/Tabs';
 
 import './App.scss';
 
 /* eslint-disable */
-const tabs = [
+const tabsFromServer = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedId: tabsFromServer[0].id,
+    selectedTitle: tabsFromServer[0].title,
```

Comment:
Store the whole tab here and then access its fields when necessary.
```suggestion
    selectedTab: tabsFromServer[0],
```